dash-renderer/src/utils/callbacks.ts

```diff
@@ -0,0 +1,8 @@
+import { omit, values } from 'ramda';
+
+import { ICallbacksState } from '../reducers/callbacks';
+import { ICallback } from '../types/callbacks';
+
+export const getPendingCallbacks = (state: ICallbacksState) => Array<ICallback>().concat(
+    ...values(omit(['stored', 'completed'], state))
```

Comment:
I take it `Array().concat(...)` is effectively `flatten`?

Reply:
`concat` allows for n-arguments, either objects or arrays - I guess 99% of the time we don't use it that way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat